src/mono/wasm/Wasm.Build.Tests/BuildPublishTests.cs

```diff
@@ -108,23 +108,25 @@ public void BuildThenPublishWithAOT(BuildArgs buildArgs, RunHost host, string id
 
             _testOutput.WriteLine($"{Environment.NewLine}Publishing with no changes ..{Environment.NewLine}");
 
+            // relinking for paths with unicode does not work:
+            // [ActiveIssue("https://github.com/dotnet/runtime/issues/83497")]
```

Comment:
Might be nice to have a FIXME here so it's easier to find by searching the codebase for them, but I see that the activeissue mentions this method needs to be updated, so it's not obligatory